package.json

```diff
@@ -69,7 +69,9 @@
         "oidc-client-ts": "3.0.1",
         "jwt-decode": "4.0.0",
         "@floating-ui/react": "0.26.11",
-        "@radix-ui/react-id": "1.1.0"
+        "@radix-ui/react-id": "1.1.0",
+        "caniuse-lite": "1.0.30001643",
+        "electron-to-chromium": "1.5.2"
```

Comment:
These are both transitive dependencies of `browserslist` but forcing resolution de-duplicates them in the bundle